AlphaWallet/Tokens/ViewModels/AddHideTokensViewModel.swift

```diff
@@ -19,10 +19,10 @@ private enum AddHideTokenSections: Int {
     }
 }
 
-struct AddHideTokensViewModel {
+class AddHideTokensViewModel {
     private let sections: [AddHideTokenSections] = [.displayedTokens, .hiddenTokens]
     private let filterTokensCoordinator: FilterTokensCoordinator
-    private var tokens: [TokenObject]
+    var tokens: [TokenObject]
```

Comment:
Looks like `tokens` can still be private

Reply:
ooh, yep you right.